diff --git a/app/classes/Framadate/Services/InputService.php b/app/classes/Framadate/Services/InputService.php
index b39e9f2..468dc9a 100644
--- a/app/classes/Framadate/Services/InputService.php
+++ b/app/classes/Framadate/Services/InputService.php
@@ -95,8 +95,8 @@ class InputService {
     }
 
     public function filterComment($comment): ?string {
-        $comment = str_replace("\r\n", "\n", $comment);
-        return $this->returnIfNotBlank((string)$comment);
+        $comment = str_replace("\r\n", "\n", (string)$comment);
+        return $this->returnIfNotBlank($comment);
     }
 
     public function validateDate(DateTime $date, DateTime $minDate, DateTime $maxDate): DateTime {
diff --git a/app/inc/smarty.php b/app/inc/smarty.php
index 2cde508..4984a37 100644
--- a/app/inc/smarty.php
+++ b/app/inc/smarty.php
@@ -29,6 +29,8 @@ $smarty->registerPlugin(\Smarty::PLUGIN_MODIFIER, '__f', '__f');
 $smarty->registerPlugin(\Smarty::PLUGIN_MODIFIER, 'constant', 'constant');
 $smarty->registerPlugin(\Smarty::PLUGIN_MODIFIER, 'max', 'max');
 $smarty->registerPlugin(\Smarty::PLUGIN_MODIFIER, 'preg_match', 'preg_match');
+$smarty->registerPlugin(\Smarty::PLUGIN_MODIFIER, 'strtotime', 'strtotime');
+$smarty->registerPlugin(\Smarty::PLUGIN_MODIFIER, 'count', 'count');
 $smarty->setTemplateDir(ROOT_DIR . '/tpl/');
 $smarty->setCompileDir(ROOT_DIR . COMPILE_DIR);
 $smarty->setCacheDir(ROOT_DIR . '/cache/');
diff --git a/tpl/page.tpl b/tpl/page.tpl
index 4387301..ce2f84c 100644
--- a/tpl/page.tpl
+++ b/tpl/page.tpl
@@ -38,7 +38,7 @@
     <script src="https://framasoft.org/nav/nav.js"></script>
 {/if}
     <div class="container ombre">
-        {include file='header.tpl'}
+        {block name="site_header"}{include file='header.tpl'}{/block}
         {block name=main}{/block}
         </main>
     </div> <!-- .container -->
diff --git a/tpl/studs.tpl b/tpl/studs.tpl
index 827638c..154e398 100644
--- a/tpl/studs.tpl
+++ b/tpl/studs.tpl
@@ -26,6 +26,8 @@
 
 {/block}
 
+{block name="site_header"}{/block}
+
 {block name=main}
 
 {if !$accessGranted && !$resultPubliclyVisible}
